.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.6.3.cjs
```

Comment:
Shouldn't this be part of this commit?

Reply:
Yes, it should. Fixed in the latest commit.